### PR TITLE
OS specific dependencies for PIP not as optional

### PR DESCRIPTION
**Pull request by richibrics**

### CODE REVIEW

**Comment by richibrics** [comment]
Tested on macOS, python3.10 and correctly installs ioturing-applesmc, pyobjc and ignores tinywintoast

**Comment by infeeeee** [comment]
Working on linux perfectly.

Don't forget to update the readme as well. If you are there, `git` is not a requirement any more on all platforms.

**Comment by richibrics** [comment]
Yes, I also added a section to quickly see the install command as it's only a pip command for those who already have python3.7+ and pip installed, so that it seems easy to install (as it really is).

File: README.md [hunk 1]
```diff
@@ -14,11 +14,22 @@ But the most important thing: **works on all OSs and all architectures ! Windows
 
 ## Install
 
-You can easily install IoTuring through pip. Although the version is os-specific, so follow the instructions below to install the right version for your operating system.
+## Who knows how it works
```
Comment (infeeeee) [inline README.md:17]: This should be `###`, now it's the same level as `## Install`, looks strange

Comment (richibrics) [reply README.md:17]: Yeah, I also checked that, don't know how I haven't noticed it :(

**Comment by richibrics** [comment]
Merging since it works on all platforms

**Pull request merged**